packages/ckeditor5-core/src/editor/editorconfig.jsdoc

```diff
@@ -161,10 +161,16 @@
  *
  *		toolbar: [ 'bold', 'italic', '|', 'undo', 'redo' ]
  *
+ *	or `'-'` to make a line break, so toolbar will become multi-line:
+ *
+ *		toolbar: [ 'bold', 'italic', '-', 'undo', 'redo' ]
+ *
+ *	Line break will work only in extended format when `shouldNotGroupWhenFull` option is set to `true`
+ *
  * * **`toolbar.viewportTopOffset`** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar.
  * Useful when a page with which the editor is being integrated has some other sticky or fixed elements
  * (e.g. the top menu). Thanks to setting the toolbar offset the toolbar will not be positioned underneath or above the page's UI.
- * * **`toolbar.shouldNotGroupWhenFull`** &ndash; When set `true`, the toolbar will stop grouping items that wrap to the next line when
+ * * **`toolbar.shouldNotGroupWhenFull`** &ndash; When set `true`, the toolbar will stop grouping items and let them wrap to the next line when
```

Comment:
ESLint shows the `max-len` error in my IDE for this line. Do you have ESLint properly configured @pkwasnik? Pre-commit hooks should block this kind of commit.

Reply:
Probably it only checks \*.js files (for them I see errors). I will need to configure that.